src/include/btree_inline.h

```diff
@@ -1559,6 +1559,10 @@ __wt_ref_addr_copy(WT_SESSION_IMPL *session, WT_REF *ref, WT_ADDR_COPY *copy)
          * page with rec_result == 0 we will free the addr being copied - possibly after the null
          * check above - and this function will attempt to copy from freed memory.
          */
+        /*
```

Comment:
This can be merged with the comment above.

Suggested reword: "WT_READ_ONCE currently takes both a variable to read and the location to save it to. Once WT-11718 is complete it will only take the variable to be read and can be used inside this assert.".